src/openforms/contrib/digid/utils.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from django.templatetags.static import static
+
+from openforms.authentication.base import LoginLogo
```

Comment:
I don't like that the `authentication` app is leaking into the generic contrib, it should be the other way around.

What about just returning the dict of kwargs that are needed, that also solves the ugly bit where you need to pass around the `label`. The plugin itself can then do something like `return LoginLogo(label=self.get_label(), **get_digid_logo(request))`